### PR TITLE
1.0.0

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -15,11 +15,4 @@ module.exports = function (grunt) {
     var bastion = require('./bastion.js');
 
     bastion(grunt);
-
-    grunt.registerTask('ci', [
-        'eslint',
-        'jshint',
-        'htmlhint',
-        'karma:ci'
-    ]);
```
Comment (waldenraines) [inline Gruntfile.js]: :question: 

Does CI only exist in katello then?

Comment (ehelms) [inline Gruntfile.js]: This was an override that only Bastion could see -- https://github.com/Katello/bastion/commit/eaf0eb0b7d8f203b003c8556b18dc08a7dcc5d3c and https://github.com/Katello/bastion/pull/53/files#diff-1a1f7ef6ca0e0de38474f61db2b3ead7R14

Comment (waldenraines) [inline Gruntfile.js]: Ah, okay, thanks.

**Comment by waldenraines** [comment]
ACK

**Pull request merged**